src/components/profiles/profileChip.vue

```diff
@@ -1,23 +1,26 @@
 <template>
-    <v-chip
-      v-if="entry.link"
-      :href="entry.link"
-      target="_blank"
-      class="google-font break-word text-none pa-2"
-      color="grey darken-3"
-      
-      :outlined="!this.$vuetify.theme.dark"
-    >
-      <v-img
-        v-show="entry.svg"
-        :src="require('@/assets/img/svg/' + entry.svg + '.svg')"
-        svg-inline
-        max-width="15"
-        max-height="15"
-        class="mr-2"
-      />
-      {{ entry.name }}
-    </v-chip>
+  <v-card
+    v-if="entry"
+    :href="entry.link"
+    target="_blank"
+    class="google-font break-word text-none pa-0 ma-0"
+    :outlined="!this.$vuetify.theme.dark"
+  >
+    <v-list class="pa-0" :color="this.$vuetify.theme.dark ? 'grey darken-3' : ''">
+      <v-list-item class="ma-0 pa-0 pr-2" dense>
+        <v-list-item-avatar class="ma-0 pa-0" width="20px" height="20px">
+          <v-img
+            v-show="entry.svg"
+            :src="require('@/assets/img/svg/' + entry.svg + '.svg')"
+            svg-inline
+            style="max-width:20px; max-height:20px;"
+            class=""
+          />
+        </v-list-item-avatar>
+        {{entry.name}}

```

Comment:
Should put the entry.name in a v-list-item-title wrapped by a v-list-item-content instead of relying on potentially changing default behaviors tho if it works, it works I guess

Reply:
It actually was wrapped in a v-list-item-content originally, but that stopped it from displaying in IE11.